docs/website/docs/getting-started.md

```diff
@@ -24,12 +24,28 @@ Install dlt using `pip`:
 pip install -U dlt
 ```
 
-The command above installs (or upgrades) the library core, in the example below we use DuckDB as a destination so let's add a `duckdb` dependency:
+Install dlt using `pixi`:
+
+```sh
+pixi add dlt
+```
+
+Install dlt using `conda`:
+
+```sh
+conda install -c conda-forge dlt
+```
+
+The commands above install (or upgrade) the core library, in the example below we
+use DuckDB as a destination so let's add a `duckdb` dependency:
 
 ```sh
 pip install "dlt[duckdb]"
 ```
 
+While the example uses `pip` for installation, users of other package managers
+should adapt the command accordingly to install the `duckdb` package.
```

Comment:
What would be an example of adapting the command for `conda`?

Reply:
@burnash I removed this part and only moved commands to `installation.md`